_opensearch/rest-api/create-index.md

```diff
@@ -0,0 +1,155 @@
+---
+layout: default
+title: Create index
+parent: REST API reference
+grand_parent: OpenSearch
+nav_order: 7
+---
+
+# Create index
+
+While you can create an index by using a document as a base, you can also just create an empty index for use later.
+
+## Example
+
+The following example demonstrates how to create an index with specifies numbers of shards and replicas, specifies that `age` is of type `integer`, and assigns a `sample-alias1` alias to the index.
```

Comment:
Typo here. "create an index with a non-default number of primary and replica shards,"